src/polycubed/src/server/Parser/YangNodes.tcpp

```diff
@@ -294,13 +308,59 @@ void Yang::ParseList(
   parent->AddChild(std::move(resource));
 }
 
+/* TODO
```

Comment:
What is this TODO (commented?)

Reply:
Oops, I forgot to remove it, I wanted to create a parseAugment but I seem to remember that there were problems with the LibYang version and also, then I realize I didn't need to parse an Augment (until now).